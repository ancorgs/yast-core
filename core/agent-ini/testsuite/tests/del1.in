+Section0
   v = xxx
   u = zzz
-Section0
+Section1
  Val1 = value11
  Val2 = value12
  +Sub1
    Vals1 = vals1
    Vals2 = vals2
  -Sub1
  +Sub2
    Vals3 = vals3
    Vals4 = vals4
  -Sub2
  Val3 = value13
-Section1

+Section2
  Val1 = value21
  Val2 = value22
  Val3 = value23
-Section2
