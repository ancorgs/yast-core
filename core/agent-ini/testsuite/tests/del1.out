(nil)
+Section1
  Val1=value11
  Val2=value12
+Sub1
  Vals2=vals2
-Sub1
  Val3=value13
-Section1
